raven/models/rv.py

```diff
@@ -309,6 +311,15 @@ def linear_transform(self, value):
 
         self._linear_transform = value
 
+    @property
+    def deaccumulate(self):
+        """Convert cumulative precipitation to precipitation rate (mm/d)."""
+        return ":Deaccumulate" if self._deaccumulate is True else ""
```

Comment:
Vraiment un nitpick pour nitpicker ici mais le `is True` n'est pas redondant?

Reply:
Je voulais m'assurer que c'est bien un boolean, et pas juste 42.